functionary/ui/forms/forms.py

```diff
@@ -0,0 +1,52 @@
+from django.forms import (
+    BooleanField,
+    CharField,
+    DateField,
+    DateTimeField,
+    DecimalField,
+    Field,
+    FloatField,
+    Form,
+    IntegerField,
+    JSONField,
+    TimeField,
+)
+
+field_mapping: dict[str, Field] = {
```

Comment:
Last type declaration comment I'll make...

This is presumably intended to be:
```python
field_mapping: dict[str, Type[Field]] = {
```

Reply:
Removed